bundles/CoreBundle/src/EventListener/Frontend/ElementListener.php

```diff
@@ -136,8 +136,9 @@ private function handleAdminUserDocumentParams(Request $request, ?Document $docu
         }
 
         // editmode document
+        $isPimcoreStudio = $request->query->getBoolean('pimcore_studio');
         if ($this->editmodeResolver->isEditmode($request)) {
-            $document = $this->handleEditmode($document, $user, $request->getSession());
+            $document = $this->handleEditmode($document, $user, $request->getSession(), $isPimcoreStudio);
```

Comment:
```suggestion
            $isPimcoreStudio = $request->query->getBoolean('pimcore_studio');
            $document = $this->handleEditmode($document, $user, $request->getSession(), $isPimcoreStudio);
```
and removing the line above isEditMode, as `$isPimcoreStudio` is not used elsewhere in the function than this

Reply:
Just moved it so that we don't have a that long line, but also fine to do it inline